en/concept/mixing.md

```diff
@@ -161,6 +163,19 @@ A mixer file must be named **XXXX._main_.mix** if it is responsible for the mixi
 > **Note** `XXXX` in the mixer file name is set in the airframe definition (e.g. see `set MIXER quad_w` in [airframes/10015_tbs_discovery](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/airframes/10015_tbs_discovery))
 
 
+### Loading a Custom Mixer {#loading_custom_mixer}
+
+PX4 loads appropriately named mixer files from the SD card directory **etc/mixers**, by preference, and then the version in Firmware.
+
+To load a custom mixer, you should give it the same name as a "normal" mixer file (that is going to be loaded by your airframe) and put it in the **etc/mixers** directory on your flight controller's SD card.
+
+Most commonly you will use the mixer file for your current airframe (**XXXX._aux_.mix**), where the `XXXX` prefix is the set in the airframe configuration file (e.g. [airframes/10015_tbs_discovery](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/init.d/airframes/10015_tbs_discovery) loads **quad_w._aux_.mix** - see `set MIXER quad_w`).
+
+**Note**
+- The airframe mixer file is preferred because it is always loaded (it is not always obvious which other mixer files are loaded, if any).
```

Comment:
The airframe mixer as opposed to what? This is not clear to me.

Reply:
As opposed to _any other mixer_ (e.g. pass.aux.mix). The problem here is that there are a bunch of other mixers and it isn't clear which of them might be loaded. The only one you *know* will be loaded is airframe. The stuff in brackets was supposed to make that clear. 

I'll revisit this after you answer https://github.com/PX4/Devguide/pull/973/files#r392770530 - might just remove the line.

Reply:
I see. You're right, the airframe mixer is the one that's easier to know which one is actually used.